routes/index.js

```diff
@@ -41,10 +41,30 @@ router.post('/register', (req, res) => {
     })
 })
 
+function __dataCheck(body) {
+    switch(body) {
+        case body.username === undefined:
+            return false
+        case body.firstName === undefined:
+            return false
+        case body.lastName === undefined:
+            return false
+        case body.email === undefined:
+            return false
+        case body.avatar === undefined:
+            return false
+        case body.bio === undefined:
+            return false
```

Comment:
You don't need previous returns, just have a list of cases and have return false on the last one